doc/src/manual/constructors.md

```diff
@@ -552,16 +564,15 @@ Therefore we want to avoid an interface that allows the user to construct instan
 `SummedArray{Int32,Int32}`. One way to do this is to provide only an outer constructor for `SummedArray`.
 This can be done using method definition by type:
 
-```julia
-type SummedArray{T<:Number,S<:Number}
-    data::Vector{T}
-    sum::S
-
-    function (::Type{SummedArray}){T}(a::Vector{T})
-        S = widen(T)
-        new{T,S}(a, sum(S, a))
-    end
-end
+```jldoctest
+julia> type SummedArray{T<:Number,S<:Number}
+           data::Vector{T}
+           sum::S
+           function (::Type{SummedArray}){T}(a::Vector{T})
```

Comment:
I'm confused by the paragraph right before this, isn't this an inner constructor?